clang/include/clang/AST/ASTContext.h

```diff
@@ -1139,6 +1139,7 @@ class ASTContext : public RefCountedBase<ASTContext> {
   CanQualType SatShortFractTy, SatFractTy, SatLongFractTy;
   CanQualType SatUnsignedShortFractTy, SatUnsignedFractTy,
       SatUnsignedLongFractTy;
+  CanQualType MFloat8Ty;
```

Comment:
For this and all related "in llvm code" naming, do you mind using the `ArmMFloat8` prefix instead of `MFloat8`?  I ask because BFloat16 has an architecture independent meaning whereas this new type is specific to Arm and its related builtins.

Reply:
I am not sure. The closest I see to a target dependent type are some openCL types
 OCLClkEventTy like OCLQueueTy.
I will change to ArmMFloat8 as you suggested.